master/getting-started/kubernetes/troubleshooting.md

```diff
@@ -53,4 +53,11 @@ However, if you do need to assign a particular address to a Pod, {{site.prodname
 - You can request an IP that is available in {{site.prodname}} IPAM using the `cni.projectcalico.org/ipAddrs` annotation.
 - You can request an IP using the `cni.projectcalico.org/ipAddrsNoIpam` annotation. Note that this annotation bypasses the configured IPAM plugin, and thus in most cases it is recommended to use the above annotation. 
 
-See the [Requesting a Specific IP address]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#requesting-a-specific-ip-address) section in the CNI plugin reference documentation for more details.
\ No newline at end of file
+See the [Requesting a Specific IP address]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#requesting-a-specific-ip-address) section in the CNI plugin reference documentation for more details.
+
+## calico/node Readiness probe failed: Bird is not ready: Unestablished peers
+
+The calico/node container may report an "unready" status in Kubernetes with this message. In most cases, this means a particular peer is unreachable in the cluster. Users should ensure BGP connectivity between the two hosts is allowed in their environment.
```

Comment:
This section is very host-centric, but this can also happen for non-full-mesh scenarios as well.

Reply:
I think we should split it like this:

"This occurs when a particular BGP peer is not reachable. This can happen either due to network connectivity issues when reaching a configured BGP peer . . . This can also happen when inactive Node resources are configured when using node-to-node mesh"

Or something along those lines.

Reply:
I'm not sure exactly what it is you are trying to fix here, so i'm having trouble filling in the ". . ."